adapter/protobuf/src/Mu/Adapter/ProtoBuf.hs

```diff
@@ -287,8 +291,13 @@ instance {-# OVERLAPS #-}
          => ProtoBridgeField sch ty ('FieldDef name ('TList t)) where
   fieldToProto (Field (FList xs))  = foldMap (oneFieldValueToProto fieldId) xs
```

Comment:
You could use the same `supportsPacking` flag to write using packed encoding as well. But you have to make sure not to use packed encoding if the field has been annotated with `[packed=false]` in the `.proto` file. I'm not sure if you have that information available at this point.

Reply:
In theory we could add a new kind of annotation about whether they should be packed or not when serializing to Protocol Buffers.

Reply:
Done! The last commit adds support for packed fields, and reads that information from the `.proto` file. If not found, it defaults to packed fields if the type supports that functionality.